src/CargoBuild/sdk/Sdk.props

```diff
@@ -0,0 +1,93 @@
+<!--
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  
+  Licensed under the MIT license.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<PropertyGroup>
+		<UsingMicrosoftCargoBuildSdk>true</UsingMicrosoftCargoBuildSdk>
```

Comment:
Is this used anywhere? Seems only read in the unit test. If it is used, should probably set this to true on Condition it's empty.

Reply:
This is a common pattern to use as a signal that the SDK was used. The .NET SDK does the same thing, and we do for NoTargets, etc as well

Reply:
Example from .NET SDK: https://github.com/dotnet/sdk/blob/41bcaa914b54c96ad20e784d0baeccf2d7ab7fa1/src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props#L22-L31

And Microsoft.NET.Sdk.Web:
https://github.com/dotnet/sdk/blob/41bcaa914b54c96ad20e784d0baeccf2d7ab7fa1/src/WebSdk/Web/Targets/Sdk.Server.props#L17

And then our NoTargets:
https://github.com/microsoft/MSBuildSdks/blob/a72501989d5b03f3f22257d41416d59695f1bd3c/src/NoTargets/Sdk/Sdk.props#L10

So IMHO, leave this property as you have it.